tests/Aspire.Hosting.Azure.Tests/AzureRedisExtensionsTests.cs

```diff
@@ -81,23 +81,10 @@ public async Task AddAzureRedisWithAccessKeyAuthentication(string? kvName)
             redis.WithAccessKeyAuthentication(builder.AddAzureKeyVault(kvName));
```

Comment:
(nit) you can get rid of line 75: `kvName = "redis-cache-kv";` now, as that is no longer necessary.